src/driver/postgres/PostgresQueryRunner.ts

```diff
@@ -197,10 +206,21 @@ export class PostgresQueryRunner extends BaseQueryRunner implements QueryRunner
     /**
      * Executes a given SQL query.
      */
-    async query(query: string, parameters?: any[]): Promise<any> {
+    async query(rawQuery: string, parameters?: any[]): Promise<any> {
         if (this.isReleased)
             throw new QueryRunnerAlreadyReleasedError();
 
+        let query: string;
+        if(this.metadata != null) {
+            const serializedMetadata = Object.entries(this.metadata).map(([key, value]) => `${key}=${JSON.stringify(value)}`).join(",");
+            const sanitizedSerializedMetadata = serializedMetadata.replace(/\*\//g, "");
+
+            query = `/* ${sanitizedSerializedMetadata} */ ${rawQuery}`;
```

Comment:
will this mess up datadog/pganalyze query tracking? hopefully not, but would be curious. Cause datadog also already injects additional comments to manage postgres tracing.

I gues we can just test on a dev stack to see

Reply:
I was going by this: https://docs.datadoghq.com/database_monitoring/guide/tag_database_statements/ I would hope they would call something out if it would break their postgres tracing